docs/orchestration/nextflow/nextflow-overview.md

```diff
@@ -370,22 +370,22 @@ aws batch submit-job \
 After submitting a workflow, you can monitor the progress of tasks via the AWS Batch console.
 For the "Hello World" workflow above you will see five jobs run in Batch - one for the head node, and one for each `Channel` text as it goes through the `hello` process.
 
-For a more complex example, you can try the following, which will run the [RNASeq workflow](https://nf-co.re/rnaseq) developed by the [NF-Core project](https://nf-co.re/) against data in the [1000 Genomes AWS Public Dataset](https://registry.opendata.aws/1000-genomes/):
+For a more complex example, you can try the following, which will run a demo [RNASeq workflow](https://github.com/nextflow-io/rnaseq-nf) against data in the [1000 Genomes AWS Public Dataset](https://registry.opendata.aws/1000-genomes/):
```

Comment:
Using the demo is ok, but there is more value in pulling from a community library like the NF-core. Was there another workflow (e.g. `sarek`) that could be used?

Reply:
I'll test sarek and swap it out if it works.